Dockerfile.nanoserver

```diff
@@ -0,0 +1,73 @@
+FROM microsoft/windowsservercore:1709 AS base
+
+SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
+
+ENV PYTHON_VERSION 3.6.5
+ENV PYTHON_HASH 9e96c934f5d16399f860812b4ac7002b
+ENV DESTINATION_FOLDER C:\\tools
+
+WORKDIR /tmp
+
+RUN $python_url = ('https://www.python.org/ftp/python/{0}/python-{0}-amd64.exe' -f $env:PYTHON_VERSION); \
+    Write-Host ('Downloading {0}...' -f $python_url); \
+    [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12; \
+    (New-Object System.Net.WebClient).DownloadFile($python_url, '/tmp/python-installer.exe'); \
+    Write-Host ('Checking hash...' -f $python_url); \
+    if ((Get-FileHash python-installer.exe -Algorithm md5).Hash -ne $env:PYTHON_HASH) { Write-Host 'Python installer checksum verification failed!'; exit 1; }; \
+    $install_folder = Join-Path -Path $env:DESTINATION_FOLDER -ChildPath 'python'; \
+    Write-Host ('Installing into {0}...' -f $install_folder); \
+    Start-Process python-installer.exe -Wait -ArgumentList @('/quiet', 'InstallAllUsers=1', 'TargetDir={0}' -f $install_folder, 'PrependPath=1', 'Shortcuts=0', 'Include_doc=0','Include_pip=1', 'Include_test=0');
+
+WORKDIR /
+
+RUN Remove-Item tmp -Recurse -Force
+
+FROM microsoft/powershell:6.0.2-nanoserver
```

Comment:
Why use this image?  Why not keep this more thin with `microsoft/nanoserver:1709`?

Reply:
'cause powershell is needed (in lieu of bash script) to build the AZ CLI

Reply:
Why not finish the build in the build layer and copy the assets to a clean layer?

Reply:
Sorry for the late answer, I completely missed your reply. Yep I'll do that right away

Reply:
Ok, it has passed some times so I needed to check the code. The answer to your question is that I'm not building the azure-cli, but just installing it, just like it happens in the original Dockerfile, where a for loop is used to install the needed packages, just as I do with the powershell script. Since it was ok for Linux I followed the same approach